### PR TITLE
Fix static typing and doc typos

### PR DESCRIPTION
**Pull request by zhuoqun-chen**
# What does this PR do?

Fixes typos:

- 1 return type doc string typo in `scheduling_ddim.py`
- 1 static type checking typo in `scheduling_ddpm.py`

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

### CODE REVIEW

File: src/diffusers/schedulers/scheduling_ddpm.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def __init__(
         sample_max_value: float = 1.0,
         timestep_spacing: str = "leading",
         steps_offset: int = 0,
-        rescale_betas_zero_snr: int = False,
+        rescale_betas_zero_snr: bool = False,
```
Comment (a-r-r-o-w) [inline src/diffusers/schedulers/scheduling_ddpm.py:197]: You might have to run `make fix-copies` to fix all instances of this type hint. If you find more inconsistencies, please feel free to open follow-up PRs!

**Review by a-r-r-o-w — APPROVED** [review]
Thank you for your first contribution! Looks correct to me, but might require a few minor changes in other files with `make fix-copies`

cc @sayakpaul

File: src/diffusers/schedulers/scheduling_ddpm.py (hunk 1 above)
Comment (zhuoqun-chen) [reply src/diffusers/schedulers/scheduling_ddpm.py:197]: Thank you @a-r-r-o-w for the information!

**Comment by yiyixuxu** [comment]
thanks for your contribution!

**Comment by zhuoqun-chen** [comment]
Thank you @a-r-r-o-w and @yiyixuxu for reviewing and merging this PR! Have a nice day!

**Pull request merged**